modules/central-services-public-config.adoc

```diff
@@ -271,52 +271,166 @@ When you disable autoscaling, the `minReplicas` and `maxReplicas` parameters do
 | The maximum number of replicas for autoscaling.
 
 | `scanner.resources.requests.memory`
-| The memory request for Scanner to override the default value.
+| The memory request for the StackRox Scanner to override the default value.
 
 | `scanner.resources.requests.cpu`
-| The CPU request for Scanner to override the default value.
+| The CPU request for the StackRox Scanner to override the default value.
 
 | `scanner.resources.limits.memory`
-| The memory limit for Scanner to override the default value.
+| The memory limit for the StackRox Scanner to override the default value.
 
 | `scanner.resources.limits.cpu`
-| The CPU limit for Scanner to override the default value.
+| The CPU limit for the StackRox Scanner to override the default value.
 
 | `scanner.dbResources.requests.memory`
-| The memory request for Scanner database deployment to override the default values.
+| The memory request for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.requests.cpu`
-| The CPU request for Scanner database deployment to override the default values.
+| The CPU request for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.limits.memory`
-| The memory limit for Scanner database deployment to override the default values.
+| The memory limit for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.limits.cpu`
-| The CPU limit for Scanner database deployment to override the default values.
+| The CPU limit for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.image.registry`
-| A custom registry for the Scanner image.
+| A custom registry for the StackRox Scanner image.
 
-| `scanner.image.name`
-| The custom image name that overrides the default Scanner image name (`scanner`).
+| ` `
```

Comment:
was this an accident or due to my "duplicate" comment? Sorry if I caused confusion. I think this should be reverted